Sources/HashTreeCollections/HashNode/_HashNode+Initializers.swift

```diff
@@ -57,7 +57,7 @@ extension _HashNode {
       childMap: .empty,
       count: 1
     ) { children, items in
-      assert(items.count == 1 && children.count == 0)
+      assert(items.count == 1 && children.isEmpty)
```

Comment:
`children` is of type `UnsafeMutableBufferPointer`. Do you have evidence that this change has a meaningful impact?

Reply:
No, I may be mistaken. Since it is a RandomAccessCollection type, I don’t think there will be any difference.